protos/sigstore_trustroot.proto

```diff
@@ -148,9 +148,11 @@ message TrustedRoot {
 // signing. In particular, it primarily contains service URLs that a Sigstore
 // signer may need to connect to for the online aspects of signing.
 message SigningConfig {
-        // MUST be application/vnd.dev.sigstore.signingconfig.v0.1+json
+        // MUST be application/vnd.dev.sigstore.signingconfig.v0.1+json or
+        // application/vnd.dev.sigstore.signingconfig.v0.2+json
         string media_type = 5;
 
+        // Deprecated: Use certificate_authority_urls
```

Comment:
Can you explain the way these deprecations are useful (compared to just removing the fields):
* if a service does not want to support the new fields, I woudl expect them to just keep serving signingconfig v0.1? 
* if a client supports signingvonfig v0.2 then they need to support the new fields so won't need these

Maybe I'm missing a case?

Reply:
is the idea that clients have an easier time supporting v0.2 if they can just update version number and keep doing what they are doing now?

Reply:
I was aiming to concurrently support both to avoid a breaking change, but you're right, this isn't necessary. 

Also, only sigstore-python and -go support signing configs and I'm fairly certain no one is using them for the Go client, so making this breaking change is likely to have no impact.